wolfssl/ssl.h

```diff
@@ -1176,6 +1179,12 @@ WOLFSSL_ABI WOLFSSL_API int  wolfSSL_connect(WOLFSSL* ssl);
 WOLFSSL_ABI WOLFSSL_API int  wolfSSL_write(
     WOLFSSL* ssl, const void* data, int sz);
 WOLFSSL_ABI WOLFSSL_API int  wolfSSL_read(WOLFSSL* ssl, void* data, int sz);
+#ifdef WOLFSSL_LEANPSK_STATIC
+WOLFSSL_API int  wolfSSL_write_inline( WOLFSSL* ssl, const void* data,
```

Comment:
The implementation of this function is in `mplabx/small-psk-build/psk-ssl.c`.  Why are these defined here?